### PR TITLE
wiki story include angular material updated

### PR DESCRIPTION
**Pull request by StefanNieuwenhuis**
docs(@angular/cli): change wiki documentation for Angular Material
change in documentation for integrating Angular Material with @angular/cli without errors.

Closes #6274

### CODE REVIEW

**Comment by sumitarora** [comment]
@StefanNieuwenhuis Please fix the commit message.

File: docs/documentation/stories/include-angular-material.md [hunk 1]
```diff
@@ -9,22 +9,29 @@ ng new my-app
...
 
 import { MaterialModule } from '@angular/material';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 // other imports
 
 @NgModule({
   imports: [
     ...
-    MaterialModule
+    MaterialModule,
```
Comment (RicardoVaranda) [inline docs/documentation/stories/include-angular-material.md]: This Module is now deprecated, material team are now asking users to declare the ngModule of the component they want to use, for example MdButtonModule

File: docs/documentation/stories/include-angular-material.md [hunk 2]
```diff
@@ -9,22 +9,29 @@ ng new my-app
...
+    BrowserAnimationsModule
```
Comment (RicardoVaranda) [inline docs/documentation/stories/include-angular-material.md]: We should also inform users of NoopAnimationsModule if they prefer not to add another dependency

**Comment by filipesilva** [comment]
@StefanNieuwenhuis is this PR up to date with the official instructions provided by Angular Material?

**Comment by StefanNieuwenhuis** [comment]
I still can't complete this pr, because the review is failing again and again. Can you please help me?

**Comment by jiverson** [comment]
@StefanNieuwenhuis your commit message needs needs to follow Commit message guidelines: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#commit

**Comment by filipesilva** [comment]
You would also need to make a single commit instead of three. You can do that by rebasing, or by making a new PR with a single commit.

**Comment by hansl** [comment]
Material has a better guide on how to integrate with CLI.

**Pull request closed without merging**